lib/src/core/implementation/signature_body_source_parameters.dart

```diff
@@ -0,0 +1,26 @@
+class SignatureBodySourceParameters {
+  final String method;
+  final Uri uri;
+  final Map<String, String> headers;
+  final String nonce;
+  final int timestamp;
+  final String clientId;
+  final List<String> signatureBodySourceComponents;
+  final List<int> body;
```

Comment:
I doubt Dart's `List<int>` to be equivalent to C#'s `byte[]`.

Reply:
Yes, that should be `Uint8List`

Reply:
https://api.flutter.dev/flutter/dart-typed_data/Uint8List-class.html